AnimatedComponents/App.tsx

```diff
@@ -99,10 +98,13 @@ const App = () => {
         thumbColors={thumbColors}
       />
       <IconTransitionButton
+        failedIcon={<CrossIcon />}
+        failedLabel="Payment Failed"
         onPress={onIconTransition}
```

Comment:
function name must justify the action its created for, onIconTrasition is not the action we are doing here, Its handlePayment or something related to payment.